src/style.css

```
@@ -0,0 +1,70 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.nav {
+  display: flex;
+  
+}
+
+.header__image {
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  list-style-type: none;
+  justify-content: space-around;
+}
+
+.nav__link {
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 100%;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__item {
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.is-active::after,
+.nav__link:active::after {
+  top: 93%;
```

```suggestion
  bottom: 0;
```
It would be better. try to avoid such weird numbers